docs/src/interfaces/stream/usingFifo.rst

```diff
@@ -122,11 +111,8 @@ Below is the equivalent code in C++
    rogue::interfaces::stream::FifoPtr fifo = rogue::interfaces::stream::Fifo::create(150, 20, false)
 
    # Connect the src and dst
-   rogueStreamConnect(src, dst);
-
-   # Add the Fifo as a stream tap
-   rogueStreamTap(src, fifo);
+   src >> dst;
 
-   # Connect the monitor to the FIfo output
-   rogueStreamConnect(fifo, mon)
+   # Add the Fifo as a second stream and on to the monitor (reverse order show as an example)
```

Comment:
maybe add a comment that the FIFO will assert back pressure on the mon or maybe change this example to a FIFO that will not back pressure on a data path (a lot of time you don't want monitors (like live displays) to back pressure the stream source